### ISSUE TITLE
In the PR list page show if a PR is blocked by another issue/PR.

### ISSUE THREAD

**Issue by Greg-NetDuma** [go-gitea/gitea#29018]
### Feature Description

As title, taken from https://github.com/go-gitea/gitea/issues/8659.

It would be useful, implemented similarly to either:
* how conflicts and review states are displayed
* Or they should have a different icon to indicate it cannot be merged until the dependencies are  resolved.

**Comment by gitZokki** [comment]
What do you think about #29117 ?

**Comment by lunny** [comment]
Is this really necessary? Or too much information in the pull request list will bother users?

**Comment by Gr3q** [comment]
Yes. I would have been ok with a different colored blocked icon on the start of the PR item if the PR depends on other open PRs
